native/src/main/java/io/ballerina/lib/data/yaml/utils/JsonTraverse.java

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
Why this file named as `JsonTraverse`?

Reply:
We are traversing over a JSON value when creating value out for a union type expected scenario. So, I guess this name is correct.

Reply:
Can't we use the `data.jsondata` as dependent for this instead of duplicating the logic?